helm-chart/renku-core/values.schema.json

```diff
@@ -209,4 +208,4 @@
     ],
     "title": "Values",
     "type": "object"
-  }
\ No newline at end of file
+  }
```

Comment:
We should add `versions` (and its properties) to the schema as well so the values file gets properly validated by helm